corehq/apps/reports/views.py

```diff
@@ -384,6 +384,8 @@ def hq_download_saved_export(req, domain, export_id):
     # quasi-security hack: the first key of the index is always assumed
     # to be the domain
     assert domain == export.configuration.index[0]
+    export.last_accessed = datetime.utcnow()
```

Comment:
it's possible that in this scenario we also want to fire off an async rebuild task